### DISCUSSION TITLE
[PL] Możliwość dodawania większej ilości bloków zaawansowanych w modułach

### DISCUSSION THREAD

**Discussion by hahyes** [YetiForceCompany/YetiForceCRM#14126] [Feature improvement]
Witam,

z tego co widzę, jest możliwość działania na jednym bloku zaawansowanym w module. Tak się składa, że potrzebuje dwóch bloków zaawansowanych na osobne dane. W edycji modułu nie znalazłem takiej możliwości. Czy planujecie wprowadzenie takiej możliwości w przyszłości? Czy jeden blok zaawansowany bazuje na tym, że dla każdego modułu jest przeznaczona tylko jedna tabela na pola?

Pozdrawiam.

**Comment by bpabiszczak** [comment]
Nigdy żaden klient nie miał takiej potrzeby, gdy się pojawi i będzie chciał za to zapłacić, wówczas dodamy. Niestety to dość skomplikowane [kilka dni pracy], więc sami z siebie czego takiego nie dodamy [mamy inne rzeczy w kolejce]. Wynika to z tego, że nikt nie ma takiej potrzeby biznesowej [gdybyśmy raz w tygodniu słyszeli od klientów, aby to dodać, wówczas mielibyśmy to w kolejce do realizacji].

**Reply by hahyes** [reply]
Zrozumiałe, dziękuję za odpowiedź.
